controller_manager/test/test_load_controller.cpp

```diff
@@ -52,12 +52,17 @@ TEST_F(TestLoadController, load1_known_controller)
     cm_->get_loaded_controllers()[0];
 
   auto node = abstract_test_controller.c->get_node();
```

Comment:
Do we need this line? It does not have any functionality....

Reply:
Yeah, we can remove it.